pallets/admin-utils/src/lib.rs

```diff
@@ -88,7 +88,7 @@ pub mod pallet {
         #[pallet::weight(T::WeightInfo::sudo_set_default_take())]
         pub fn sudo_set_default_take(origin: OriginFor<T>, default_take: u16) -> DispatchResult {
```

Comment:
Should this be renamed to reflect the changed functionality?
```suggestion
        pub fn sudo_set_max_delegate_take(origin: OriginFor<T>, max_take: u16) -> DispatchResult {
```